src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/StrandArtifactUnitTest.java

```diff
@@ -195,4 +195,72 @@ public void testHighAlleleFraction() throws IOException {
 
     }
 
+    @Test
+    // Asserting that when the annotation is run without the "TLOD" key on the vc then the annotation does nothing
+    public void testMissingTumorLod() throws IOException {
```

Comment:
It's probably worth extracting the code shared with other unit tests in this class, or otherwise shortening this test, especially since it is just checking for no error when TLOD is missing

Reply:
I chose to shorten the test, because the other two tests looked non-trivial to extract compared to the amount of code it would save.